### PR TITLE
fix: only print status message when explicitly provided in `Actor.exit()`

### PR DESCRIPTION
**Pull request by B4nan**
The default (generic) status message from `Actor.exit` was always overriding the final status message from basic crawler (that has much more information). This removes the default messages from there, keeping the one from basic crawler.

Sidenote: maybe we should also add current concurrency to the status messages?

### CODE REVIEW

**Comment by B4nan** [comment]
Hmm, apparently, the codeowners thing won't add multiple people (even in a public repo).

**Comment by B4nan** [comment]
> Hmm, apparently, the codeowners thing won't add multiple people (even in a public repo).

So it should be a single line I guess.

https://stackoverflow.com/questions/66821568/require-github-pr-approvals-from-multiple-groups

**Review by metalwarrior665 — APPROVED** [review]

**Review by mnmkng — APPROVED** [review]

**Pull request merged**
